demo/clean-kubernetes.sh

```diff
@@ -5,6 +5,8 @@ set -aueo pipefail
 # shellcheck disable=SC1091
 source .env
 
+bin/osm mesh uninstall -f --mesh-name "$MESH_NAME" --namespace "$K8S_NAMESPACE"
```

Comment:
Should we add a `make build-osm` before running the uninstall command?

Reply:
We could but unnecessary in the context of current usage (always invoked via demo script). Is it ok to assume the cleanup script for the demo is run post install, at which time the cli binary is built already? I don't see value in even running this without the demo being run prior.